### PR TITLE
test: Fix ceph-objectstore-tool usage check

### PR DESCRIPTION
**Pull request by dzafman**

### CODE REVIEW

File: qa/standalone/special/ceph_objectstore_tool.py [hunk 1]
```diff
@@ -1030,7 +1030,7 @@ def main(argv):
 
     # Specify a bad --op command
     cmd = (CFSD_PREFIX + "--op oops").format(osd=ONEOSD)
-    ERRORS += test_failure(cmd, "Must provide --op (info, log, remove, mkfs, fsck, export, export-remove, import, list, fix-lost, list-pgs, rm-past-intervals, dump-journal, dump-super, meta-list, get-osdmap, set-osdmap, get-inc-osdmap, set-inc-osdmap, mark-complete)")
+    ERRORS += test_failure(cmd, "Must provide --op (info, log, remove, mkfs, fsck, repair, export, export-remove, import, list, fix-lost, list-pgs, rm-past-intervals, dump-journal, dump-super, meta-list, get-osdmap, set-osdmap, get-inc-osdmap, set-inc-osdmap, mark-complete)")
```
Comment (amitkumar50) [inline qa/standalone/special/ceph_objectstore_tool.py]: i believe tracker or description would been good

**Comment by tchaikov** [comment]
"special" is not exercised by any test suite so far, so just merge it.

**Comment by dzafman** [comment]
@tchaikov Yes, the special directory is not run by teuthology.  But you can run it locally by using the run-standalone.sh script.

# from build directory
../qa/run-standalone.sh ceph_objectstore_tool.py

runs this one test.  Or without any arguments to run all tests in qa/standalone except qa/standalone/special/test-failure.sh

**Comment by tchaikov** [comment]
@dzafman thanks, i knew it. just wanted to put it here for the posterity. also could you help review #17747?

**Pull request merged**